docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
++
+
+.Example `rhoas_service_account` resource
+[source]
+----
+resource "rhoas_service_account" "my-service-account" {
+  name        = "my-service-account"
+  description = "<description of service account>"
+}
+
+output "client_id" {
+  value = rhoas_service_account.my-service-account.client_id
+}
+
+output "client_secret" {
+  value     = rhoas_service_account.my-service-account.client_secret
+  sensitive = true
+}
+----
++
+The values for the client ID and client secret will be set owhen you run `terraform apply`.
+. Enter the example `rhoas_topic` resource to create a Kafka topic with default values. This example uses the `topic` identifier and creates the `my-topic` Kafka topic. As you have already created the Kafka instance, Terraform can check dependencies for this new topic resource and knows the Kafka ID when you run this example resource.
++
+.Example `rhoas_topic` resource with default values
+[source]
+----
+resource "rhoas_topic" "topic" {
+		name = "my-topic"
+		partitions = 1
+		kafka_id = rhoas_kafka.instance.id
+	}
+
+----
++
+. Enter the `rhoas_acl` resource to create an ACL binding. This example uses the `"acl"` identifier. The following list is a description of each field in an ACL binding resource.
+
+* `resource_type`: The type of resource you want access to. This example uses *“TOPIC”*.
+* `resource_name`: the name of resource you want access to. This example uses the name that is passed when creating the topic.
+* `principal`: the user that this binding applies to. This example uses the service account client ID.
+* `operation_type`: The type of operation that is allowed for the given user on this resource.
+* `permission_type`: Whether permission is given or taken away.
+
++
+.Example `ACL binding` resource
+[source]
+----
+resource "rhoas_acl" "acl" {
+  kafka_id = rhoas_kafka.instance.id
+  resource_type = "TOPIC"
+  resource_name = "my-topic"
+  pattern_type = "LITERAL"
+  principal = rhoas_service_account.my-service-account.client_id
+  operation_type = "ALL"
+  permission_type = "ALLOW"
+}
+
+----
+. Save your changes.
+. Open a terminal window and enter the following command:
++
+[source,shell]
+----
+$ terraform init
+----
+. Enter the following command:
++
+[source,shell]
+----
+$ terraform apply
+----
+. Terraform displays a message that `rhoas_kafka.my-instance`, `rhoas_acl.acl`,`rhoas_service_account.my-service-account`, and `rhoas_acl.acl` will be created and displays all the field values for each resources, including fields that will be known after apply such as `client_id` and `client_secret`.
+When you're ready to create your instance, service account, topic, and set your permissions, type *Yes*.
```

Comment:
"When you're ready to create your instance, service account, **and** topic, as well as setting your permissions, type *Yes*."